Mathlib/SetTheory/Ordinal/Enum.lean

```diff
@@ -116,6 +136,10 @@ theorem enumOrd_zero : enumOrd s 0 = sInf s := by
   rw [enumOrd]
   simp [Ordinal.not_lt_zero]
 
+theorem enumOrd_eq_zero (hs : 0 ∈ s) : enumOrd s 0 = 0 := by
```

Comment:
Can this be turned into an iff?

Reply:
No, since `enumOrd Ø 0 = 0`. I did realize this lemma wasn't very confluent with `enumOrd s 0 = sInf s`, so I instead added `csInf_eq_bot_of_bot_mem` as a lemma on CC lattices.